From 392f5ca271b72c4bd2936e3f5ac519bf4ec60b09 Mon Sep 17 00:00:00 2001
From: Alejandro Mery <amery@geeks.cl>
Date: Fri, 2 Nov 2012 19:03:33 +0000
Subject: [PATCH 925/944] Revert "arm: sun4i: lower PLL6 clock to fix GPU
 stalls"

This reverts commit 1813b92cda31d9b045fdedaeb31cf6ee71c1a001.

kills SATA to fix mali, bad idea.

Signed-off-by: Alejandro Mery <amery@geeks.cl>
---
 arch/arm/mach-sun4i/clock/clock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-sun4i/clock/clock.c b/arch/arm/mach-sun4i/clock/clock.c
index 3b9a2b6..2619c6d 100644
--- a/arch/arm/mach-sun4i/clock/clock.c
+++ b/arch/arm/mach-sun4i/clock/clock.c
@@ -180,7 +180,7 @@ int clk_init(void)
 
 	/* initiate PLL6 */
 	tmpSclk = &ccu_sys_clk[AW_SYS_CLK_PLL6];
-	tmpSclk->clk->rate  = 960000000;
+	tmpSclk->clk->rate  = 1200000000;
 	tmpSclk->set_clk(tmpSclk->clk);
 	tmpSclk->clk->onoff = AW_CCU_CLK_ON;
 	tmpSclk->set_clk(tmpSclk->clk);
-- 
1.8.0

